src/main/java/sirius/kernel/commons/Files.java

```diff
@@ -28,6 +29,18 @@ public class Files {
 
     private static final Pattern NON_PATH_CHARACTERS = Pattern.compile("[^a-zA-Z0-9\\-.]");
 
+    /**
+     * Contains a list of file names and endings which are considered to be metadata.
+     */
+    private static final List<String> METADATA_FILES = List.of(
```

Comment:
Should these be Patterns like `.*\.ini`, ... to check more specifically? Some of them are prefixes, some are suffixes, some are entire file names.